lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/NativeRender.vcxproj

```diff
@@ -81,23 +81,16 @@
       <PreprocessorDefinitions>PEW_EOS_EXPORT;OVERLAY_DLL_NAME="$(OverlayDllName)";STEAM_API_DLL="$(SteamApiDll)";SDK_DLL_NAME="$(SdkDllName)";CONFIG_DIRECTORY="$(ConfigDirectory)";_DEBUG;NATIVERENDER_EXPORTS;_WINDOWS;_USRDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <PreBuildEvent>
-      <!-- When debugging, make sure to copy the EOSSDK Dll to the output directory -->
-      <Command>
-        @echo Copying "$(OutputUnityAssetsDirectory)$(SdkDllName)" to "$(TargetDir)"
-        copy "$(OutputUnityAssetsDirectory)$(SdkDllName)" "$(TargetDir)" /Y
-      </Command>
+      <!-- Execute the prebuild powershell command -->
+      <Command>powershell -NoProfile -ExecutionPolicy Bypass -File "$(ProjectDir)scripts\prebuild.ps1" -Configuration "$(Configuration)" -OutputUnityAssetsDirectory "$(OutputUnityAssetsDirectory)" -EOS_SDK_DLL_NAME "$(EOS_SDK_DLL_NAME)" -STEAM_API_DLL_NAME "$(STEAM_API_DLL_NAME)" -TargetDir $(TargetDir) -SteamLibraryDirectory "$(SteamLibraryDirectory)" -SolutionDir "$(SolutionDir)"</Command>
```

Comment:
I feel like this will fail to run on some computers... but probably not our own! Powershell often has to be installed or agreed to, in order to work, but I'm willing to deal with that for our own native libs.

Reply:
> If Steam is missing, will this still compile?

Good question - I think it might fail because it cannot copy the steam dll if it does not exist. I'll go fix that.

Reply:
I have updated the implementation to warn that the steam dll could not be copied if it doesn't exist - which will make the build succeed on systems that do not have steamworks setup.